src/plugins/intel_gpu/src/kernel_selector/cl_kernels/resample_pil_ref.cl

```diff
@@ -0,0 +1,251 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
nit: 2023

Reply:
Do you want me to do the following change?
```suggestion
// Copyright (C) 2023 Intel Corporation
```

Reply:
yes

Reply:
Updated copyright notice in all new sources